docs/MODULES.md

```diff
@@ -247,6 +248,38 @@ This module is used for decoding a QR in image (if present).
            .addSteps('decode-qr',options)
            .run()
 ```
+
+## dither-module
+
+This module approximates a color from a mixture of other colors when the required color is not available, creating illusions of the color that is not present actually.
+
+## Usage
+
+```js
+  sequencer.loadImage('PATH')
+           .addSteps('dither',options)
+           .run()
+```
+
+## draw-rectangle-module
+
+This module helps to draw a rectangle on the image and also helps to change the thickness and color.
```

Comment:
This module draws a rectangle with a starting and ending corner with the specified thickness and color of border